### PR TITLE
Added Shape Theming support to codelabs

### PR DESCRIPTION
**Pull request by yarneo**
Closes https://github.com/material-components/material-components-ios/issues/6880

### CODE REVIEW

File: MDC-104/ObjectiveC/Starter/Shrine/Shrine/ApplicationScheme.h [hunk 1]
```diff
@@ -30,6 +31,7 @@
 - (instancetype)init NS_UNAVAILABLE;
 
 @property(nonnull, readonly, nonatomic) MDCSemanticColorScheme *colorScheme;
+@property(nonnull, readonly, nonatomic) MDCShapeScheme *shapeScheme;
```
Comment (romoore) [inline MDC-104/ObjectiveC/Starter/Shrine/Shrine/ApplicationScheme.h]: If this is in the 104 starter, shouldn't 103 Complete also be changed in this PR?

Comment (yarneo) [reply MDC-104/ObjectiveC/Starter/Shrine/Shrine/ApplicationScheme.h]: 103 doesn't consist of any preliminary shape stuff, and that is only added in the starter of 104.

Comment (romoore) [reply MDC-104/ObjectiveC/Starter/Shrine/Shrine/ApplicationScheme.h]: I see, so it just sort of appears "by magic" between the codelabs.

**Comment by romoore** [comment]
Gent ping to @yarneo .  Can this be submitted after merging in the formatting changes from `master`?

**Pull request merged**